source/fundamentals/connection/connection-options.txt

```diff
@@ -73,6 +73,28 @@ parameters of the connection URI to specify the behavior of the client.
      - Specifies whether to force dispatch **all** operations to the host
        specified in the connection URI.
 
+   * - **enableUtf8Validation**
+     - boolean
+     - ``true``
+     - 
+       Specifying ``true`` enables UTF-8 validation for the 
+       connection. MongoDB throws an error when 
+       it attempts to convert data that contains invalid 
+       UTF-8 characters. The validation adds validation processing 
+       overhead since it needs to check the data.
+
+       Specifying ``false`` disables UTF-8 validation for the 
+       connection. MongoDB does not throw errors when 
+       data contains invalid UTF-8 data. If you disable validation, 
+       your application avoids the validation processing overhead 
+       but cannot guarantee consistent presentation of invalid 
+       UTF-8 data. **Editing data** while validation is disabled
+       can result in loss of data. Disabling UTF-8 validation is a 
```

Comment:
is it accurate that disabling utf8 validation can result in loss of data?  the driver will still successfully insert the data, it just won't be queryable by a client that has utf8 validation enabled.

Reply:
Here we were concerned about people reading their data (with U+FFFD replacement characters), then writing it again (or some modified version of it) as an insert/update, which will lead to the U+FFFD replacement character being written to the database instead of the original information (i.e. the original bytes in the database are no longer available afterwards)